ads/google/imaVideo.js

```diff
@@ -896,11 +896,13 @@ export function onAdLoad(global) {
 
 /**
  * Called intermittently as the ad plays, allowing us to display ad counter.
- * @param {!Object} global
+ * @param {!Object} unusedEvent
  * @visibleForTesting
  */
-export function onAdProgress(global) {
-  const {adPosition, totalAds} = global.getAdData();
+export function onAdProgress(unusedEvent) {
```

Comment:
Shall we remove the `unusedEvent` from code?

Reply:
I kept it so that it's obvious it takes an `event`. I think the minifier removes this, anyway.